sound/soc/sof/intel/hda-loader.c

```diff
@@ -92,6 +93,16 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
 		goto err;
 	}
 
+	/* DSP is powered up, set all SSPs to slave mode */
+	for (i = 0; i < chip->ssp_count; i++) {
+		snd_sof_dsp_update_bits_unlocked(sdev, HDA_DSP_BAR,
+						 chip->ssp_base_offset
+						 + i * SSP_DEV_MEM_SIZE
+						 + SSP_SSC1_OFFSET,
+						 SSP_SET_SLAVE,
+						 SSP_SET_SLAVE);
+	}
+
```

Comment:
please double-check that it's the right time to set the SSP mode to slave

Reply:
@plbossart Yes, this is the right place: after power on and before the DSP bootup. When DSP init finished the registers can't be set from the host.

From The spec : 
After power on, if the SSP input I/O clock is in high state, first transition of the clock from high
to low may be missing due to clock gating logic.